core-web/apps/dotcms-ui/src/app/shared/models/container/dot-container.model.ts

```diff
@@ -4,20 +4,20 @@ export enum CONTAINER_SOURCE {
 }
 
 export interface DotContainer {
-    archived: boolean;
+    archived?: boolean;
     categoryId?: string;
     deleted?: boolean;
-    live: boolean;
-    working: boolean;
-    locked: boolean;
+    live?: boolean;
+    working?: boolean;
+    locked?: boolean;
     friendlyName?: string;
     path?: string;
-    identifier: string;
-    name: string;
-    type: string;
+    identifier?: string;
+    name?: string;
+    type?: string;
     title?: string;
-    source: CONTAINER_SOURCE;
-    parentPermissionable: {
+    source?: CONTAINER_SOURCE;
+    parentPermissionable?: {
```

Comment:
Why all are non-required?

Reply:
I added optional here because some test cases failed due to some parameters not being passed in test cases in the canary branch.